src/app/controllers/components/ControllerHeaderForms/AddController/AddController.tsx

```diff
@@ -22,6 +24,8 @@ export const AddController = ({ clearHeaderContent }: Props): JSX.Element => {
       </p>
       <CodeSnippet
         blocks={[
+          { code: `sudo apt-add-repository ppa:maas/${minorVersion}` },
+          { code: "sudo apt update" },
```

Comment:
```suggestion
          { code: `sudo add-apt-repository ppa:maas/${minorVersion}` },
```

apt-add-repository is the old name (and a symlink to `add-apt-repository`)

as of about Bionic (or even earlier?) you don't need to `apt update` after adding a repo like this.